src/server/data-access/advisorDescriptor/index.ts

```diff
@@ -170,9 +43,10 @@ const responseProcessor = (data: typeof advisorData) => data;
 const advisorDescriptor: ServiceDescriptor = {
   templates: {
     advisor: {
-      service: ServiceNames.demo,
+      service: ServiceNames.advisor,
       responseProcessor,
       path: '/',
+      request: getAdvisorData,
       mock: getMock,
```

Comment:
Just curious, is the mock supposed to be fallback? Or going to be removed later?

Reply:
mock is used in a local development environment. Not an ideal ATM.

Something to work on in the future (probably proxy)